docs/openstack/backend_services_deployment.md

```diff
@@ -127,9 +128,13 @@ podified OpenStack control plane services.
   account passwords from the original deployment:
 
   ```
+<<<<<<< HEAD
   oc set data secret/osp-secret "AodhPassword=$AODH_PASSWORD"
   oc set data secret/osp-secret "CeilometerMeteringSecret=$CEILOMETER_METERING_SECRET"
   oc set data secret/osp-secret "CeilometerPassword=$CEILOMETER_PASSWORD"
+=======
+  oc set data secret/osp-secret "BarbicanPassword=$BARBICAN_PASSWORD"
+>>>>>>> 5bd34c5 (Barbican adoption procedure)
```

Comment:
Here's a rebase artifact, and the CI failure looks like Barbican failed to start, but other than that the patch LGTM.

Reply:
Need to set the KEK here too.  As of recent commits, this is possible as well.  This absolutely must be the same as what is in the existing barbican config or you will not be able to decrypt existing secrets.